spec/requests/api/v1.1/portfolios_controller_spec.rb

```diff
@@ -102,6 +102,11 @@
 
         expect(response).to have_http_status(:forbidden)
       end
+
+      it "uses a custom message" do
+        post "#{api_version}/portfolios", :headers => default_headers, :params => valid_attributes
+        expect(json["errors"].first["detail"]).to eq("You are not authorized to create this portfolio")
```

Comment:
@eclarizio we have this helper
https://github.com/RedHatInsights/catalog-api/blob/9a81e18dea373c2735682218921071eb97d62711/spec/support/requests_spec_helper.rb#L14